infra/growth/src/util/fixtures.js

```diff
@@ -157,6 +158,20 @@ async function createTestData() {
     currency: 'OGN',
     rewardStatus: enums.GrowthCampaignRewardStatuses.NotReady
   })
+
+  await db.GrowthCampaign.upsert({
+    id: 9,
```

Comment:
I think this should be id 10 since 9 is already use above to upsert the september rules ?

Reply:
Nice catch thanks @franckc . Fix here: 
https://github.com/OriginProtocol/origin/pull/3655